lib/puppet/provider/package/rpm.rb

```diff
@@ -15,6 +21,9 @@
   has_feature :virtual_packages
   has_feature :install_only
 
+  RPM_VERSION       = Puppet::Util::Package::Version::Rpm
+  RPM_VERSION_RANGE = Puppet::Util::Package::Version::Range
```

Comment:
This are no longer used

Reply:
I think they should be moved to `lib/puppet/provider/package/yum.rb`. It seems the only place where they are still used.